ui/apps/platform/src/Components/CompoundSearchFilter/attributes/image.ts

```diff
@@ -0,0 +1,52 @@
+import { SearchFilterAttribute } from '../types';
+
+// If you're adding a new attribute, make sure to add it to the "imageAttributes" array as well
```

Comment:
RE: My earlier comment about reducing the utils and additional code flows.

If we exported the object from these files directly, the attributes could likely be inlined which would make this comment and the need for the comment obsolete. Although I suppose the objects could be inlined even using the array approach.

Reply:
Yeah, I realized that too. If we export each attribute in the file, we probably don't need that `[entity]Attributes` object either. So instead of having

```
export const imageAttributes = { Name, OperatingSystem, Tag, Label, Registry } as const;
```

We can export `Name`, `OperatingSystem`, `Tag`, `Label`, and `Registry`. But, it could be useful to have this when we need to use all the attributes. It would prevent developers from accidentally missing one when they intended to display all. But it's debatable whether it provides enough value. I'm ok with removing this too if its preferrable.

Reply:
Good call, I think the flexibility of exporting each attribute when needed outweighs the benefits of having a single export that we filter later. 👍